Demo-InputSearchMockServer/src/pages/Homepage/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Search from '../../components/molecules/Search';
+import UserList from '../../components/organisms/UserList';
+import { Container } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+const StyledContainer = styled(Container)({
+  marginTop: '20px',
+});
+
+type User = {
+  id: number;
+  name: string;
+};
+
+const Home: React.FC = () => {
+  const [searchQuery, setSearchQuery] = useState<string>('');
+  const [users, setUsers] = useState<User[]>([]);
+
+  useEffect(() => {
+    const fetchUsers = async () => {
```

Comment:
You can move this out side the useEffect for better readability.

Reply:
resolved.